mujoco_warp/_src/io.py

```diff
@@ -742,6 +742,7 @@ def _collision_sensor_check(sensor_type, sensor_id, geom_type, err_msg):
     mapM2M=wp.array(mjm.mapM2M, dtype=int),
     # warp only fields:
     nsensortaxel=sum(mjm.mesh_vertnum[mjm.sensor_objid[mjm.sensor_type == mujoco.mjtSensor.mjSENS_TACTILE]]),
+    nsensorcontact=np.sum(mjm.sensor_type == mujoco.mjtSensor.mjSENS_CONTACT),
```

Comment:
can we now annotate `sensor_contact_adr` as having size `nsensorcontact`?

Reply:
done